zha/zigbee/endpoint.py

```diff
@@ -204,7 +204,7 @@ async def _execute_handler_tasks(
         results = await gather(*tasks, return_exceptions=True)
         for cluster_handler, outcome in zip(cluster_handlers, results):
             if isinstance(outcome, Exception):
-                cluster_handler.debug(
+                cluster_handler.warning(
```

Comment:
unrelated change? assume this was for quick visual in log during development?

Reply:
In the reconfigure dialog we show the results of the calls with success / failure. I need to update the joining UI to do something similar. It has the capability to flag failures w/ color and messaging it just isn't working atm.

Reply:
@TheJulianJES we should never generically hide base Exception types. That hides programming errors (as i found out the hard way). Error in programming caused an attribute error that was hard to track down due to this.

Its fine to swallow know errors if we know their cause.

Reply:
Ps. I move this out separate.